<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelationWidget</class>
 <widget class="QWidget" name="RelationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constraints</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Parse</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="parseConstraintPair">
        <property name="text">
         <string>constraint pair</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="parseConstraintGroup">
        <property name="text">
         <string>constraint group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="parseGlobalSymm">
        <property name="text">
         <string>global reflection symmetry</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Trace</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="traceConstraintsAuto">
        <property name="text">
         <string>trace constraints auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="traceConstraints">
        <property name="text">
         <string>trace constraints by hand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QCheckBox" name="bWithSymm">
           <property name="text">
            <string>with global symm</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="symmWeight">
           <property name="toolTip">
            <string comment="0-1" extracomment="weights between it &amp; constaint (pair &amp; group)"/>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="text">
            <string>0.7</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="bWithGroups">
              <property name="text">
               <string>with groups</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLineEdit" name="groupWeight">
              <property name="statusTip">
               <string comment="1-10" extracomment="weights of constraint group relative to pair constraint"/>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
              <property name="text">
               <string>4.0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="bWithPairs">
              <property name="text">
               <string>with pairs</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="pairWeight">
              <property name="statusTip">
               <string comment="1-10" extracomment="weights of constraint pair relative to group constraint"/>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>130</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
